modules/custom/az_digital_asset_library/az_digital_asset_library.info.yml

```diff
@@ -2,7 +2,7 @@ name: 'Quickstart Digital Asset Library'
 type: module
 description: 'The connector for the centralized University of Arizona Digital Asset Library (IntelligenceBank DAM).'
 core_version_requirement: ^9 || ^10
-package: 'The University of Arizona - Experimental'
+package: 'The University of Arizona'
```

Comment:
Look into issues on quickstart.arizona.edu with this module

Reply:
I don't recall what the issue was and I was able to embed a DAL photo just fine on a draft QS page. @camikazegreen Do you have any more specifics I should look into?

Reply:
The issue was related to the login screen not using the included config to pre-fill with the appropriate settings to login. 
I'm also ok with considering this to be a site specific issue that does not need to be resolved to move this module to stable. I haven't seen it happen to any other sites.